### PR TITLE
Added assignment, bitwise, and bit shift operator

### PR DESCRIPTION
**Pull request by Kuiprux**
I think I messed up the commit list. Sorry for that.

### CODE REVIEW

**Comment by wysohn** [comment]
Can you also possibly write some unit testing codes?

Tests that are just enough to cover the new implementations; it doesn't have to be super complex.

Refer to https://www.guru99.com/white-box-testing.html if you are not familiar with white-box testing.

**Comment by wysohn** [comment]
examples here: https://github.com/TriggerReactor/TriggerReactor/tree/master/core/src/test/java/io/github/wysohn/triggerreactor/core/script

**Review by gerzytet — APPROVED** [review]
To be honest, I'm not certain why I'm still on the reviewers list after all this time.
My concern isn't the code.  Unless you do something truly deplorable, there aren't many ways to make Interpreter.java worse than it already is.  It even looks like you've cleaned part of it up.
I'm just concerned that advanced features like bitwise operators aren't useful for TriggerReactor's community of beginners.  It can't hurt, but I'm afraid almost no-one will use it.

**Comment by gerzytet** [comment]
Wait, I glossed over the "assignment operators" part.  That's been a requested feature for a while.  Thanks for finally implementing it!

**Comment by Kuiprux** [comment]
I agree with you. I will focus more on the essential functions.
Thank you for your opinion!

File: core/src/main/java/io/github/wysohn/triggerreactor/core/script/interpreter/Interpreter.java [hunk 1]
```diff
@@ -609,20 +609,78 @@ private Integer interpret(Node node) throws InterpreterException {
...
 
                 if (isVariable(left)) {
                     left = unwrapVariable(left);
                 }
 
-                if (node.getToken().value.equals("+")
+                if ("+".equals(tokenValue)
                         && (left.type == Type.STRING || right.type == Type.STRING)) {
                     stack.push(new Token(Type.STRING, String.valueOf(left.value) + right.value, node.getToken()));
+                } else if ("&".equals(tokenValue) || "^".equals(tokenValue) || "|".equals(tokenValue)) {
```
Comment (wysohn) [inline core/src/main/java/io/github/wysohn/triggerreactor/core/script/interpreter/Interpreter.java:628]: I think from here to line 683 isn't covered by the test

**Review by wysohn — CHANGES_REQUESTED** [review]
Wow thank you for the amazing work!

As gerzytet said, you are making great impacts

Can you please also cover those parts whenever gets the chance?

**Review by wysohn — APPROVED** [review]

**Pull request merged**